### PR TITLE
Lesson 1_8

### PR DESCRIPTION
**Pull request by verafes**
Connect to Airtable API
Fetch Data from Airtable

### CODE REVIEW

**Review by AnnaPestova1 — CHANGES_REQUESTED** [review]
Hi Vera! Great work. I mentioned small issue related your comment in the question section: `POST method didn't work for me, until I changed title: title -> title: todo.title`

Here is your todo: 
image

Here is the explanation from our assignment (read it carefully and you will figure out what is the issue)
image

**Comment by verafes** [comment]
I updated POST method to use Airtable API to create new todo.

File: src/AddTodoForm.jsx [hunk 1]
```diff
@@ -14,10 +14,7 @@ function AddTodoForm({ onAddTodo }) {
     // Function to handle form submission
     const handleAddTodo = (event) => {
         event.preventDefault();
-        const newTodo = {
-            id: Date.now(),
-            title: todoTitle,
-        };
+        const newTodo = todoTitle
```
Comment (AnnaPestova1) [inline src/AddTodoForm.jsx]: now this line is unnecessary, you can put todoTitle  in onAddTodo function: onAddTodo(todoTitle);

**Review by AnnaPestova1 — APPROVED** [review]
Hi Vera! Great changes.

File: src/AddTodoForm.jsx (hunk 1 above)
Comment (verafes) [reply src/AddTodoForm.jsx]: I removed unnecessary variable. Thank you!

**Pull request merged**